sample-input/tiny-lattice/tiny-lattice.py

```diff
@@ -56,10 +53,13 @@
 
 log.py_printf('NORMAL', 'Creating cells...')
 
-cells = []
-cells.append(CellBasic(universe=1, material=uo2_id))
-cells.append(CellBasic(universe=1, material=water_id))
-cells.append(CellFill(universe=0, universe_fill=2))
+cells = list()
```

Comment:
For these super simple sample input files, with these changes, there isn't really a need to put cells and surfaces in a list.  It would be more intuitive for new users to just see the cell objects have their own names.

That said, this probably doesn't need to be included in the current PR, I might even volunteer to revamp a few of these in another PR.